### ISSUE TITLE
The test won't run

### ISSUE THREAD

**Issue by mattheman2** [ianzapolsky/baseball-simulator#1]
Everytime I try to paste in a URL link the error message pops up saying one of my URL'S are wrong

**Comment by ianzapolsky** [comment]
Hi there @mattheman2, thanks for the comment. Unfortunately this was a small pet project I built a loooong time ago, and since then it's seen zero time or investment. I'm sorry to say that the simulator itself is built in a very fragile way that depends on web scraping data from a website. The website I was scraping from at the time has no doubt undergone numerous redesigns by this point, rendering the original code I wrote totally worthless. It would take a lot of time and attention to get this project back into a functioning state, so what I'd recommend if you're trying to stand up your own baseball simulator is to use this implementation as a guide while building the actual simulation of the game, but rewrite the data layer from scratch.
